ansible_ai_connect/ai/api/model_pipelines/wca/pipelines_onprem.py

```diff
@@ -173,6 +176,19 @@ def self_test(self):
         raise NotImplementedError
 
 
+@Register(api_type="wca-onprem")
+class WCAOnPremRoleGenerationPipeline(WCAOnPremPipeline, WCABaseRoleGenerationPipeline):
+
+    def __init__(self, inference_url):
+        super().__init__(inference_url=inference_url)
+
+    def invoke(self, params: RoleGenerationParameters) -> RoleGenerationResponse:
```

Comment:
See my earlier comment about moving the "dummy" implementation from the "base" class.